src/js/components/sharedComponents/checkbox/AccordionCheckboxPrimary.jsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * CheckboxOnePrimary.jsx
```

Comment:
need to rename to match the component name

Reply:
renamed